src/core/stats/remote-metrics.ts

```diff
@@ -72,8 +74,14 @@ export class RemoteMetrics {
     }, {} as Record<string, string>)
 
     formatted['library'] = 'analytics.js'
-    formatted['library_version'] =
-      `next-${process.env.VERSION}` ?? 'next-undefined'
+
+    const type = getVersion()
+    if (type === 'web') {
+      formatted['library_version'] = `next-${version}` ?? 'next-undefined'
```

Comment:
nit, but I think `next-${version}` will always be defined, so the `?? `next-undefined` never happens, and we can safely remove it